src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -404,6 +404,26 @@ public static class MsalError
         /// </summary>
         public const string CannotInvokeBroker = "cannot_invoke_broker";
 
+        /// <summary>
+        /// MSAL is not able to find a suitable account to use with broker
+        /// </summary>
+        public const string NoBrokerAccountFound = "no_broker_account_found";
```

Comment:
qns: are these all android broker specific? if so, should that be included in the error?

Reply:
I think so, ill update